include/constants/metatile_labels.h

```diff
@@ -85,5 +85,10 @@
 // gTileset_82D504C. Below metatile probably needs generic name
 #define METATILE_MtEmber_CaveEntrance  0x346
 
+// gTileset_82D4BFC
```

Comment:
idk if this or the tileset naming pr is gonna be merged first, just make sure to resolve these comments

Reply:
Yeah will do either way. The existing ones are resolved by the tileset PR but the active PRs add some new metatitle names